### PR TITLE
Adding travis

### PR DESCRIPTION
**Pull request by samelhusseini**
- [x] Run build on all packages
- [x] Run lint on all packages
- [x] Run test on all packages

@rachel-fenichel, @moniika, @alschmiedt anything else?

### CODE REVIEW

**Comment by samelhusseini** [comment]
@rachel-fenichel I believe I have to check this in to get travis to go online. I've enabled only the build for now, will enable lint and test in separate PRs.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
```
Comment (rachel-fenichel) [inline .travis.yml:4]: 7 is not a good version--odd-numbered releases are unsupported after 6 months.  I think we should be testing on 10 and 12, which are the current active releases.

Comment (samelhusseini) [reply .travis.yml:4]: Done.

**Comment by samelhusseini** [comment]
Looks like travis got enabled :)
I'll continue working on the travis configuration in this PR. Changing to WIP.

**Comment by samelhusseini** [comment]
And we're passing, build test and lint. I think this is good for now.

**Pull request merged**